### PR TITLE
show list publisher

### PR DESCRIPTION
**Pull request by phuc021**

### CODE REVIEW

**Comment by phuc021** [comment]
ready

File: resources/views/partials/footer.blade.php [hunk 1]
```diff
@@ -1,7 +1,7 @@
     <!-- Footer section -->
     <footer class="footer_section">
         <div class="container">
-            <a href="#" class="footer_logo"><h1>book(logo)</h1>
+            <a href="#" class="footer_logo"><h1>Library Management</h1>
```
Comment (quoctd-0607) [inline resources/views/partials/footer.blade.php:4]: i18n

File: resources/views/publishers/publisher.blade.php [hunk 2]
```diff
@@ -28,40 +27,28 @@
...
-                <tr>
-                    <td scope="row">1</td>
-                    <td scope="row"><a class="book_detail" href="">Gao</a></td>
-                    <td scope="row"><a class="book_detail" href="">Gao Supper </a></td>
-                    <td scope="row"><a class="book_detail" href="">Gao Supper</a></td>
-                    <td>
-                        <button class="btn_edit" title="Edit" type="submit" value="Edit">{{ trans('publishers/publisher.edit') }}</button>
-                        <button class="btn_delete" title="Delete" type="submit" value="Delete">{{ trans('publishers/publisher.delete') }}</button>
+                        <button class="btn_edit edit_user" title="Edit" type="submit" value="Edit"><a href="{{ url('publisher/'.$data_publisher->id.'/edit')}}">{{ trans('publishers/publisher.edit') }}</a></button>
+                        <form action="{{ url("publishers/$data_publisher->id") }}" method="POST">
```
Comment (quoctd-0607) [inline resources/views/publishers/publisher.blade.php:43]: dùng route thay cho url

**Comment by phuc021** [comment]
ready

**Pull request merged**